### PR TITLE
The account type is now read and displayed

### PR DESCRIPTION
**Pull request by annando**
In one of the last pull requests we started to transmit the account type. Now we are reading it. Additionally there is now a centralized function that displays the account type. It replaces the code on multiple places with a single function.

### CODE REVIEW

File: include/Contact.php [hunk 1]
```diff
@@ -683,4 +683,47 @@ function formatted_location($profile) {
...
+/**
+ * @brief Returns the account type name
+ *
+ * The function can be called with either the user or the contact array
+ *
+ * @param array $contact contact or user array
+ */
+function account_type($contact) {
+
+	// There are several fields that indicate that the contact or user is a forum
```
Comment (rabuzarus) [inline include/Contact.php]: I know I'm really annoying but I would prefer that you write 1 - 2 more sentences which describe why we have that much fields so it wouldn't be only understandable for insiders which are familiar with the friendica history

Comment (annando) [inline include/Contact.php]: Yeah, you are :-)

**Comment by annando** [comment]
@rabuzarus Satisfied? ;-) BTW: I have the feeling that Friendica developers are people who wake up early ...

**Comment by rabuzarus** [comment]
> Satisfied? ;-)

Explanation is OK. And I will merge it. But there one thing which I don't seem ideal to me. This is the fact that this function does output translated strings. This isn't optimal for further processing because the output depends on the language. I would love to have something like an array with è.g. `label` => 'news', output => 'Neuigkeiten'`
This could be helpful e.g. at this place: https://github.com/friendica/friendica/blob/master/view/theme/frio/templates/contact_template.tpl#L65

Or for friendica internal processing.

> BTW: I have the feeling that Friendica developers are people who wake up early ...
Or couldn't sleep :-)

**Comment by annando** [comment]
In the future we could additionally transmit the value - not only the name, okay.

**Pull request merged**